modules/nf-core/glimpse2/phase/meta.yml

```diff
@@ -0,0 +1,102 @@
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
+      type: files
+      description: |
+        Either multiple BAM/CRAM files containing low-coverage sequencing reads
+        Or one VCF/BCF file containing the genotype likelihoods.
+        When using BAM/CRAM the name of the file is used as samples name.
+      pattern: "*.{bam,cram,vcf,vcf.gz,bcf,bcf.gz}"
+
+  - input_index:
+      type: file
+      description: Index file of the input BAM/CRAM/VCF/BCF file.
+      pattern: "*.{bam.bai,cram.crai,vcf.gz.csi,bcf.gz.csi}"
+
+  - input_region:
+      type: string
+      description: Target region used for imputation, including left and right buffers (e.g. chr20:1000000-2000000).
+      pattern: "chrXX:leftBufferPosition-rightBufferPosition"
+
+  - output_region:
+      type: string
+      description: Target imputed region, excluding left and right buffers (e.g. chr20:1000000-2000000).
+      pattern: "chrXX:leftBufferPosition-rightBufferPosition"
+
+  - reference:
+      type: file
+      description: Reference panel of haplotypes in VCF/BCF format.
+      pattern: "*.{vcf.gz,bcf.gz}"
+
+  - reference_index:
+      type: file
+      description: Index file of the Reference panel file.
+      pattern: "*.{vcf.gz.csi,bcf.gz.csi}"
+
+  - map:
+      type: file
+      description: File containing the genetic map.
+      pattern: "*.gmap"
+
+  - samples_file:
+      type: file
+      description: |
+        File with sample names and ploidy information.
+        One sample per line with a mandatory second column indicating ploidy (1 or 2).
+        Sample names that are not present are assumed to have ploidy 2 (diploids).
+        GLIMPSE does NOT handle the use of sex (M/F) instead of ploidy.
+      pattern: "*.{txt,tsv}"
+
+  - fasta_reference:
+      type: file
+      description: |
+        Faidx-indexed reference sequence file in the appropriate genome build.
+        Necessary for CRAM files
+      pattern: "*.fasta"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - phased_variants:
+      type: file
+      description: |
+        Output VCF/BCF file containing genotype probabilities (GP field),
+        imputed dosages (DS field), best guess genotypes (GT field),
+        sampled haplotypes in the last (max 16) main iterations (HS field) and info-score.
+      pattern: "*.{vcf,bcf,vcf.gz,bcf.gz}"
+
+  - stats_coverage:
+      type: file
+      description: Coverage statistic generate when BAM/CRAM inputs are used.
```

Comment:
Maybe also state here that this file is optional and how to get it :)

Reply:
In the example I found on Nf-Core it seems to be only mentioned in the description.
Is it okay for you ?
For the part how to get it, it seems to be automatically generated when using either a BAM or a CRAM file.

Reply:
Okay done !